src/index.js

```diff
@@ -1,7 +1,45 @@
 import './styles/style.css';
+import $ from 'jquery';
 import wrapContainer from './modules/WrapContainer.js';
 import wrapscores from './modules/wrapScores.js';
 
 wrapContainer();
 
-wrapscores();
\ No newline at end of file
+// ZEIiDb0Iy2wNzVD068jm
+
+const inputName = document.getElementById('name');
+const inputScore = document.getElementById('scoreIn');
+const submitBtn = document.getElementById('Submit');
+const refreshBtn = document.getElementById('refresh');
+
+submitBtn.addEventListener('click', () => {
+  const score = {
+    user: inputName.value,
+    score: inputScore.value,
+  };
+  $.post('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/ZEIiDb0Iy2wNzVD068jm/scores/', score, (data, status) => {
+    console.log(data);
+  }).catch((error) => {
+    console.log(error);
+  });
+});
+
+refreshBtn.addEventListener('click', () => {
+  $.get('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/ZEIiDb0Iy2wNzVD068jm/scores/', (response) => {
+    console.log(response.result);
+    const arr = response.result;
+
+    wrapscores(arr);
+  }).catch((error) => {
+    console.log(error);
+  });
+});
```

Comment:
- [ ] Please use `async` and `await` JavaScript features to consume the APIs instead of jquery features, This is a major requirement of the project. Learn about `async` and `await` [Here](https://javascript.info/async-await).